src/docker-it/run_docker_sshd.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# requires command line argument with path to rultor root folder
+echo "Path to rultor root folder is $1"
+if ($1); then
+    echo "Required path to rultor root folder"
+    exit 1
+fi
+dockerit=$1/src/docker-it
+container=rultor_sshd
+image="${container}_img"
+echo Image=$image
+echo Container=$container
+echo "Grand 600 permissions for keys"
+chmod 600 $dockerit/id_rsa
+chmod 600 $dockerit/id_rsa.pub
+echo "Building docker image " $image
+docker build -t $image $dockerit
+echo removing docker container $container
+docker stop $container
+docker rm $container
+echo "Running docker container $container"
+docker run --name $container -d -p 22 $image
+echo "Exporting container port"
+rm $1/target/sshd.port
```

Comment:
@vkuchyn we should have double quotes around each argument to rm, docker and chmod to prevent world splitting